python/bind_operators.h

```diff
@@ -20,6 +20,28 @@ void bind_comparison(pybind11::class_<T, Ignored...> &c) {
       py::call_guard<py::gil_scoped_release>());
 }
 
+template <class Other, class T, class... Ignored>
+void bind_variable_comparison(pybind11::class_<T, Ignored...> &c) {
+  c.def(
+      "__eq__", [](T &a, Other &b) { return variable::equal(a, b); },
```

Comment:
I can't see where the *existing* `__eq__` and `__ne__` (pointing to `operator==`) is removed, for all cases (variables, data arrays, datasets)?